src/server.ts

```diff
@@ -51,7 +52,7 @@ connection.onInitialize(
 		storageService = new StorageService();
 		scannerService = new ScannerService(storageService, settings);
 
-		const files = await findFiles('**/*.scss', {
+		const files = await findFiles('**/*.{scss,vue}', {
```

Comment:
🚫 I think this change needs to be made configurable through settings, because not all extension users use Vue.

Reply:
We can't import Vue file SCSS part in other SCSS or Vue file,
So I think we don't need scan Vue files.

I will remove it.